invenio_requests/services/requests/results.py

```diff
@@ -57,6 +57,11 @@ def links(self):
         """Get links for this result item."""
         return self._links_tpl.expand(self._identity, self._request)
 
+    @links.setter
+    def links_tpl(self, links):
```

Comment:
minor: change the parameter name to `links_tpl` to make sure it is a template and not a dict, and avoid issues with subsequent calls to `links` (would fail to expand a dict)
```suggestion
    def links_tpl(self, links_tpl):
```